test-app/tests/helpers/container-query/assert-dimensions.ts

```diff
@@ -0,0 +1,39 @@
+import { find } from '@ember/test-helpers';
+
+type Dimensions = {
+  height: number;
+  width: number;
+};
+
+export function assertDimensions(assert: Assert, dimensions: Dimensions): void {
```

Comment:
Not only is this function easier to type (before, `CustomAssert` had tried to append to the `assert` type that `qunit-dom` already appends to `qunit`), the tests seem to now finish sooner (~35 seconds before, ~23 seconds now).